src/rustinfo.rs

```diff
@@ -85,29 +85,42 @@ fn load_setup(rust_info: &mut RustInfo) {
                     }
                 }
 
+                let mut triple = String::new();
+
                 let mut value = values.remove("target_arch");
-                if value.is_some() {
-                    rust_info.target_arch = Some(value.unwrap());
+                if let Some(value) = value {
+                    triple.push_str(&value);
```

Comment:
what if target_arc is none? why not default to unknown?

Reply:
Done.